### PR TITLE
[curielogger] Move kibana's index pattern creation to Go

### PR DESCRIPTION
**Pull request by flaper87**
Complete the migration of ES configs to curielogger. Next step is adding
the dashboards creation

### CODE REVIEW

File: curiefense/curielogger/curielogger/outputs.elasticsearch.go [hunk 1]
```diff
@@ -362,6 +373,49 @@ func (l ElasticsearchLogger) getESClient() *elasticsearch.Client {
...
+	for i := 0; i < 60; i++ {
+
+		rst, err := client.Do(req)
+
+		if err == nil && rst.StatusCode == 200 {
+			log.Printf("[INFO]: kibana index pattern created %s", kbUrl)
+			break
+		}
+
+		if rst.StatusCode == 409 {
```
Comment (xavier-rbz) [inline curiefense/curielogger/curielogger/outputs.elasticsearch.go:409]: rst can be `nil`, this results in an error (running on helm):
```
2021/02/25 21:24:04 GRPC server listening on :9001
2021/02/25 21:24:04 [INFO] Logstash logging routine started
2021/02/25 21:24:04 [INFO] Prometheus (prometheus) metrics updating routine started
2021/02/25 21:24:04 [DEBUG]: configuring kibana
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xff5729]

goroutine 38 [running]:
main.(*ElasticsearchLogger).ConfigureKibana(0xc000040100)
        /curielogger/curielogger/outputs.elasticsearch.go:409 +0x609
created by main.(*ElasticsearchLogger).Configure
        /curielogger/curielogger/outputs.elasticsearch.go:525 +0x4cc
```

File: curiefense/curielogger/curielogger.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ outputs:
         overwrite: true
         use_data_stream: true
         url: "http://elasticsearch:9200/"
+        kibana_url: "http://curiefense:5601"
```
Comment (xavier-rbz) [inline curiefense/curielogger/curielogger.yml:9]: Shouldn't this be http://**kibana**:5601 ?

Comment (xavier-rbz) [reply curiefense/curielogger/curielogger.yml]: fixed in https://github.com/curiefense/curiefense/pull/168/commits/ab8f8d0e1e4f4222287a94b260686b5c453d93e1

File: curiefense/curielogger/curielogger/outputs.elasticsearch.go (hunk 1 above)
Comment (flaper87) [reply curiefense/curielogger/curielogger/outputs.elasticsearch.go:409]: mmh, does this happen when the `KibanaUrl` option is not set?

Comment (flaper87) [reply curiefense/curielogger/curielogger/outputs.elasticsearch.go:409]: Ah, nvm! This happens when err is not nil

**Review by xavier-rbz — APPROVED** [review]

**Pull request merged**